entities/FireAleph.java

```diff
@@ -0,0 +1,29 @@
+package entities;
+
+public class FireAleph extends Magic {
+	String name = "Fire א";
+	String description = "A basic spell that sets enemies ablaze. Some enemies will be particularly damaged by fire, for example creatures of ice.";
+			
+	int itemID = -1001;
+	
+	int mpCost = 1;
+	
+	void use(PlayerCharacter player) {
+		// Have say not usable
+	}
+	
+	void useInBattle(PlayerCharacter player, Opponent enemy) {
+		if(mpCheck(player)) {			
+	  	  if(enemy.type == "grass" || enemy.type == "ice") {
+		  	enemy.hp -= 3;
+		  } else if(enemy.type == "water") {
+		  	enemy.hp -= 0;
```

Comment:
Redundant line. Maybe say, after the initial if, else if (enemy.type != water) {enemy.hp -= 1} and then nothing else?

Reply:
Oh that's big facts